composer.json

```diff
@@ -16,7 +16,9 @@
 	"require-dev": {
 		"phpunit/phpunit": "^9",
 		"nextcloud/coding-standard": "^1.0.0",
-		"nextcloud/ocp": "dev-master"
+		"nextcloud/ocp": "dev-master",
+		"vimeo/psalm": "^5.18",
+		"sabre/dav": "4.5.0"
```

Comment:
you use it directly?

Reply:
Yes: https://github.com/nextcloud/ldap_contacts_backend/blob/main/lib/Model/Card.php#L28-L32